### PR TITLE
Fixes 'invalid-name' (C0103) warnings.

### PR DESCRIPTION
**Pull request by pponnuvel**

### CODE REVIEW

**Review by mustafakemalgilor — APPROVED** [review]
LGTM :shipit:

File: tests/unit/test_kernel.py [hunk 1]
```diff
@@ -113,7 +113,7 @@ def setUp(self):
 
 class TestKernelCallTraceManager(TestKernelBase):
 
-    def test_CallTraceManager_handler(self):
+    def test_call_trace_manager_handler(self):
```
Comment (dosaboy) [inline tests/unit/test_kernel.py:116]: test_calltrace_manager_handler

File: tests/unit/test_openvswitch.py [hunk 2]
```diff
@@ -101,14 +101,14 @@ def setUp(self, *args, **kwargs):
 
 class TestCoreOpenvSwitch(TestOpenvswitchBase):
 
-    def testBase_offload_disabled(self):
+    def testbase_offload_disabled(self):
```
Comment (dosaboy) [inline tests/unit/test_openvswitch.py:104]: test_base_offload_disabled

File: tests/unit/test_openvswitch.py [hunk 3]
```diff
@@ -101,14 +101,14 @@ def setUp(self, *args, **kwargs):
...
         enabled = OpenvSwitchBase().offload_enabled
         self.assertFalse(enabled)
 
     @utils.create_data_root({('sos_commands/openvswitch/ovs-vsctl_-t_5_get_'
                               'Open_vSwitch_._other_config'):
                              '{hw-offload="true", max-idle="30000"}'})
-    def testBase_offload_enabled(self):
+    def testbase_offload_enabled(self):
```
Comment (dosaboy) [inline tests/unit/test_openvswitch.py:111]: test_base_offload_enabled

**Review by dosaboy — COMMENTED** [review]
couple nits otherwise lgtm (sorry for slow review)

**Pull request merged**